R/sharpen.Rmd

```diff
@@ -0,0 +1,98 @@
+---
+title: "Untitled"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## GitHub Documents
+
+This is an R Markdown format used for publishing markdown documents to GitHub. When you click the **Knit** button all R code chunks are run and a markdown file (.md) suitable for publishing to GitHub is generated.
+
+## Including Code
+
+You can include R code in the document as follows:
+
+```{r 1.1 conv2d}
+# library(png)
+
+convolve2d <- function(input, filter) {
+  nrow <- dim(input)[1]
+  ncol <- dim(input)[2]
```

Comment:
Maybe add some comments in your function since it is quite long. 

```suggestion

convolve2d <- function(input, filter) {
  nrow <- dim(input)[1]
  ncol <- dim(input)[2]
```

Reply:
will add some comments.